apps/experiments/models.py

```diff
@@ -275,6 +275,12 @@ class AgentTools(models.TextChoices):
     SCHEDULE_UPDATE = "schedule-update", gettext("Schedule Update")
 
 
+class ExperimentStatus(models.TextChoices):
+    DRAFT = "Draft", "Draft"
+    RELEASED = "Released", "Released"
+    DEPRECATED = "Deprecated", "Deprecated"
```

Comment:
nit: use lowercase for slugs

```suggestion
    DRAFT = "draft", "Draft"
    RELEASED = "released", "Released"
    DEPRECATED = "deprecated", "Deprecated"
```